### PR TITLE
UI bug fixes

### PR DESCRIPTION
**Pull request by B-Tanonaka**
1. Button colors specified so that they look the same across different browsers
2. Dropdown in the camera page no longer hides behind the navbar
3. Navbar fixed so that it is not in the center of the registration screen on ios safari
4. Loading spinner added when an image is sent to the AI and is waiting for the predicted category

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -8,15 +8,15 @@ import {
...
-import LandingPage from './Components/LandingPage';
-import CameraCapture from './Components/Camera/CameraCapture';
-import Leaderboard from './Components/Leaderboard';
-import Profile from './Components/Profile/Profile';
-import SuccessfulSubmission from './Components/SuccessfulSubmission/SuccessfulSubmission';
-import Register from './Components/Register';
-import Login from './Components/Login';
-import PageNotFound from './Components/PageNotFound';
-import Navbar from './Components/Navbar';
+import LandingPage from './components/LandingPage';
```
Comment (JMStudiosJoe) [inline client/src/App.jsx:11]: i see the lowercase change here but the folder seems to still be uppercase, should the folders be then lowercase?

File: client/src/utils/URLpath.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
+export const aiEndpoint = process.env.REACT_APP_AI_KEY || 'https://aii.littersort.ai';
+
 export default function URLpath(endpoint, extension) {
-  const deployed = process.env.REACT_APP_API_KEY;
+  const deployed = process.env.REACT_APP_API_KEY || 'https://api.littersort.ai';
```
Comment (JMStudiosJoe) [inline client/src/utils/URLpath.js:4]: if not set just ping prod? hmmmm seems risky imo easy setup for being attacked

Comment (JMStudiosJoe) [reply client/src/utils/URLpath.js:4]: I could be overthinking it, just marking here for now, no need to fix until confirmed and can be in another pr

**Review by JMStudiosJoe — COMMENTED** [review]
couple of questions but great fixes

File: client/src/App.jsx (hunk 1 above)
Comment (B-Tanonaka) [reply client/src/App.jsx:11]: I did not realize casing on Github does not change is the letters are all the same. I manually changed it for the components and mockData folders.

File: client/src/utils/URLpath.js (hunk 2 above)
Comment (B-Tanonaka) [reply client/src/utils/URLpath.js:4]: There were issues connecting the env file to the frontend and this was the workaround to get everything running for now. This is a temporary solution and we will remove the direct links once we figure out what is going wrong.

Comment (B-Tanonaka) [reply client/src/utils/URLpath.js]: The direct links are no longer needed so they have been removed.

**Pull request merged**